compiler/src/test/scala/org/scalajs/core/compiler/test/JSSAMTest.scala

```diff
@@ -49,7 +48,6 @@ class JSSAMTest extends DirectTest with TestHelpers {
       def foo(x: Int): Int
     }
 
-    @ScalaJSDefined
```

Comment:
This change provokes some more line number changes below: at lines [61 and 64](https://github.com/gzm0/scala-js/blob/7edfee83c3b576aa993fe8f21f9a9b6378714c0f/compiler/src/test/scala/org/scalajs/core/compiler/test/JSSAMTest.scala#L61-L66), the line numbers should be 15 and 16, respectively.